src/main/webapp/script.js

```diff
@@ -73,19 +78,53 @@ function linkListElement(url) {
   return liElement;
 }
 
-/** TODO: Create Pagination for Dashboard*/
-
 /** Retrieves the previous page */
 function previous() {  // eslint-disable-line
-  /**   TODO */
+  try {
+    if (currentPage == 1) throw new Error('This is the first page');
+    currentPage--;
+    addThreads();
+  } catch (err) {
+    console.log(err.message);
+  }
 }
 
 /** Retrieves the next page */
 function next() {  // eslint-disable-line
-  /**   TODO */
+  try {
+    if (currentPage == numberOfPages) throw new Error('This is the last Page');
```

Comment:
Do you want to throw an error, or just disable the button to change page instead?